examples/eks-cluster-with-argocd/main.tf

```diff
@@ -0,0 +1,184 @@
+terraform {
```

Comment:
Should we create a **gitops** folder under example and move this example under that? This will allow us to add other Gitops examples as well here

Reply:
I think Argo is critical enough that it should be top level example